.github/workflows/node.js.yml

```diff
@@ -17,7 +12,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: lts/*
```

Comment:
Changed this because of the line above `name: Active LTS`, but I'm thinking that might just be more boilerplate from the GitHub Actions wizard, rather than an intentional name

Reply:
Let's keep this hard-coded as "16" -- I've found that it's better for us to bump the Node.js version ourselves!